.gitlab-ci.yml

```diff
@@ -317,13 +318,46 @@ check-dependent-cumulus:
     DEPENDENT_REPO: cumulus
     CARGO_UPDATE_CRATES: "sp-io polkadot-runtime-common"
 
+#### stage:                        build
```

Comment:
just looked into an entire pipeline, this one is better be named `publish`, it doesn't build much

Reply:
and this comment moved beneath the following job too